_data/errors.yml

```diff
@@ -0,0 +1,93 @@
+oidc:
+  auth:
+    - title: Bad client_id
+      id: bad-client-id
+      accordion-id: bad-client
+      content: |
+        ###### Why it's happening
+        The issuer (also known as the client id) provided does not match a service provider registered in the IdP
+        ###### What to do:
+        - Ensure the authentication request has a `client_id` value
```

Comment:
These instructions in the "what to do" sections also need periods.

Reply:
fixed in https://github.com/18F/identity-dev-docs/pull/382/commits/ec23adf4753192c2c0a61891f6594a25ae604f70